src/internal/connector/support/m365Support.go

```diff
@@ -71,3 +73,14 @@ func CreateListFromBytes(bytes []byte) (models.Listable, error) {
 
 	return list, nil
 }
+
+func HasAttachments(body models.ItemBodyable) bool {
+	if body.GetContent() == nil || body.GetContentType() == nil ||
+		*body.GetContentType() == models.TEXT_BODYTYPE || len(*body.GetContent()) == 0 {
+		return false
+	}
+
+	content := *body.GetContent()
+
+	return strings.Contains(content, "src=\"cid:")
```

Comment:
Is this the right check? Should't this check for the image tag primarily = `<img`?

Reply:
The recommendations I've seen are to look for the CID tag and not the 'img' type. I assume there can be other types?

Reply:
Hmmm ok. Wasn't sure if the CID portion is guaranteed to be there.